ci_framework/roles/tempest/ansible/ansibleee.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Apply OperatorGroup manifest
+  kubernetes.core.k8s:
```

Comment:
this requirement has to be added in the /requirements.yml file, we already have most of the needs there